core/spec.md

```diff
@@ -2698,11 +2674,18 @@ The following describes the attributes of Registry model:
     it is valid for an implementation to only support one (`1`) Version when
     `maxversions` is set to `0`.
   - When the limit is exceeded, implementations MUST prune Versions by
-    deleting the oldest Version (based on creation times) first, skipping the
-    Version marked as "default". An exception to this pruning rule is if
-    `maxversions` value is one (`1`) then the newest Version of the Resource
-    MUST always be the "default" and the `setdefaultversionsticky` aspect
-    MUST be `false`.
+    deleting the oldest Version first, skipping the
+    Version marked as "default". The strategy for finding the oldest Version
+    to delete is as follows:
+    1. Find all Versions that are roots (i.e. `ancestor` is the same as its
+       `versionid`)
+    1. If many exist, find the Version with the oldest `createdat` date
+    1. If many exist, find the lowest Version according to alphabetical order
+       of the `versionid` attribute
+    Once the single oldest Version is determined, delete it.
+    An exception to this pruning rule is if `maxversions` value is one (`1`)
```

Comment:
I know this was here before your PR, but it doesn't read correctly to me now...
What do you think about:
A special case for the pruning rules is that if `maxversions` value is one (`1`), then the "default" Version is not skipped, which means it'll be deleted and the new Version will become "default".

Or...

above change: ...skipping the Version marked as "default".
to: ...skipping the Version marked as "default" if `maxversions` is not set to one (`1`).

Thoughts?

Reply:
Adding the first suggestion, it's a bit more verbose but I think it will be more clear to readers.